src/scenes/AcceleratorRouter/Modules/ModuleView.tsx

```diff
@@ -44,7 +45,7 @@ export default function ModuleView(): JSX.Element {
   const { activeLocale } = useLocalization();
   const contentRef = useRef(null);
   const { moduleId } = useParams<{ moduleId: string }>();
-  const { getModule, module, deliverables } = useGetModule();
```

Comment:
Not sure if it is related but I am seeing duplicated modules in the list view

Reply:
same for deliverables tab

Reply:
Not sure why. Will review

Reply:
I will fix that separately. It is not related to events.